src/components/MainContent.jsx

```diff
@@ -8,100 +9,132 @@ import HouseJoy from "../assets/Group 8.png";
-              <div>
-                <TextContent
-                  text=" Divine favour is my advantage; my reality is shaped by God's favour. "
-                  className="text-[22px]  quicksand font-[300]"
-                />
-                <TextContent
-                  text="Malachi 1:2-3, Romans 9:11-13"
-                  className="my-1 text-[18px] linearColor"
-                />
+              <div className="w-full flex gap-1  max-w-1/2  h-fit ">
+                <div>
+                  <TextContent
+                    text={confession2 || "no default text"}
+                    className="text-[22px]  quicksand font-[300]"
+                  />
+                  <TextContent
+                    text={scripture2 || "no default text"}
+                    className="my-1 text-[18px] linearColor"
+                  />
+                </div>
               </div>
-            </div>
-            <div className="w-full flex gap-1  max-w-1/2  h-fit ">
-              <div className="py-3 px-[3px]  h-fit linear"></div>
-              <div>
-                <TextContent
-                  text=" I enter my next level by God's favour. I scale heights at record speed. "
-                  className="text-[22px] quicksand font-[300]"
-                />
-                <TextContent
-                  text="Psalm 44:3"
-                  className="my-1 text-[18px] linearColor"
-                />
+              <div className="w-full flex gap-1   h-fit ">
+                <div>
+                  <TextContent
+                    text={confession3 || "no default text"}
+                    className="text-[22px] quicksand font-[300]"
+                  />
+                  <TextContent
+                    text={scripture3 || "no default text"}
+                    className="my-1 text-[18px] linearColor"
+                  />
+                </div>
               </div>
             </div>
-          </div>
 
-          {/* Prayer Section */}
-          <div className="linear w-1/2 px-4 py-4 rounded-[30px] mb-4">
-            <Title text="PRAYER" className="text-[16px] px-4 p-2 text-center" />
-            <div className="space-y-3">
-              <div>
-                <TextContent
-                  text="~ Lord, I thank you for your favour at work in my life."
-                  className="text-[22px] quicksand font-[300]"
-                />
-                <TextContent
-                  text="(Gen 39:21)"
-                  className="text-[18px] quicksand"
-                />
-              </div>
+            {/* Prayer Section */}
+            <div className="linear w-1/2 px-4 py-4 rounded-[30px] mb-4">
+              <Title text="PRAYER" className="text-[16px] px-4 text-center" />
+              <div className="">
+                <div>
+                  <TextContent innerHtml={insertLineBreaks(prayer)} />
+                </div>
 
-              <div>
+                {/* <div>
                 <TextContent
```

Comment:
Get rid of this code, or leave a comprehensive comment header, describing why you have it